src/run/mod.rs

```diff
@@ -0,0 +1,48 @@
+use args::RunSubcommands;
+
+use crate::{
+    build::ensure_web_setup,
+    external_cli::{cargo, wasm_bindgen, CommandHelpers},
+    manifest::package_name,
+};
+
+pub use self::args::RunArgs;
+
+mod args;
+mod serve;
+
+pub fn run(args: &RunArgs) -> anyhow::Result<()> {
+    let cargo_args = args.cargo_args_builder();
+
+    if let Some(RunSubcommands::Web(web_args)) = &args.subcommand {
+        ensure_web_setup()?;
+
+        // If targeting the web, run a web server with the WASM build
+        println!("Building for WASM...");
+        cargo::build::command().args(cargo_args).ensure_status()?;
+
+        println!("Bundling for the web...");
+        wasm_bindgen::bundle(&package_name()?, args.profile())?;
+
+        let port = web_args.port;
+        let url = format!("http://127.0.0.1:{port}");
```

Comment:
I've had situations where `https` on `localhost` was necessary for development of a project. Do you think it would be nice to have the option here?

Reply:
In my experience, setting up HTTPS is not worth it for a development server such as this. The primary purpose of TLS is to secure communication across untrusted networks, but `127.0.0.1` is completely local, so there's no network to worry about. `bevy run web` is not meant for production. In those cases you build everything ahead of time and serve it with something like Nginx, which implements TLS already.

Reply:
Is it possible to create a valid certificate for `localhost` to enable HTTPS?
I'd say that when we need it, we can create a follow-up issue to enable that functionality :)

Reply:
Yes, assuming openssl or the like is installed.